packages/SwingSet/package.json

```diff
@@ -56,7 +55,7 @@
     "@endo/zip": "^0.2.34",
     "ansi-styles": "^6.2.1",
     "anylogger": "^0.21.0",
-    "better-sqlite3": "^8.2.0",
+    "better-sqlite3": "^9.1.1",
```

Comment:
I see discussion of some pretty fine details about sqlite among kernel devs. @gibson042 are you confident that this upgrade is OK? Or should it get review by others?